modules/sdk-coin-bera/test/unit/bera.ts

```diff
@@ -302,7 +302,7 @@ describe('Bera', function () {
 
       await basecoin
         .verifyTransaction({ txParams, txPrebuild, wallet, verification })
-        .should.be.rejectedWith('txPrebuild should only have 1 recipient but 2 found');
```

Comment:
Update code owners for this file. 

Wallets team does not own the coin specific tests.

Reply:
Added 
/modules/sdk-coin-bera/ @BitGo/ethalt-team